specification/appendix/commitment_discounts.md

```diff
@@ -0,0 +1,278 @@
+# Commitment Discounts
+
+A commitment discount is a billing discount model that offers reduced rates on preselected SKUs in exchange for an obligated usage or spend amount over a predefined term. Commitment discounts typically consist of a set of purchase and usage records within cost and usage datasets.
+
+Usage-based commitment discounts obligate a customer to a total quantity of time of a preselected term, typically measured in "Hours".  In some cases, usage-based commitment discounts allow similar resources of different size classifications to relatively benefit from commitment discount rates.  This is typically known as "size flexibility", and is determined by multiplying the number of actual hours consumed by a resource by its predetermined normalization factor to derive the number of "Normalized Hours" consumed.
```

Comment:
While I am not willing to hold up the approval of this PR, I do think that this document will strongly benefit from either 1) presenting an example of how size flexibility surfaces in the data, or 2) explicitly stating that a deeper explanation of size flexibility is beyond the scope of this document.

Reply:
Agree, would like to iterate to that goal too.

Reply:
Size Flexibility Appendix: https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/535